subsys/bluetooth/Kconfig

```diff
@@ -10,6 +10,7 @@ menuconfig BT
 	# will need some refactoring to work on SMP systems.
 	depends on !SMP
 	select NET_BUF
+	select KERNEL_WARN_PENDING
```

Comment:
Same question here, why isn't this opt-in? And even if enabled by default, you should `imply` to allow it to be turned off too.

Reply:
Done.